src/ubx_message.toit

```diff
@@ -105,15 +108,15 @@ class Message:
     sub-class.
   */
   constructor.from_reader reader/BufferedReader:
-    if (reader.byte 0) != 0xb5 or (reader.byte 1) != 0x62: throw INVALID_UBX_MESSAGE_
+    if (reader.byte 0) != 0xb5 or (reader.byte 1) != 0x62: throw INVALID_UBX_HEADER_
 
     // Verify the length and get full the packet.
     length ::= (reader.byte 4) | (((reader.byte 5) & 0xff) << 8)
-    if not 0 <= length <= 512: throw INVALID_UBX_MESSAGE_
+    if not 0 <= length <= 512: throw INVALID_UBX_SIZE_
     frame ::= reader.bytes length + 8
 
     // Verify the checksum.
-    if not is_valid_frame_ frame: throw INVALID_UBX_MESSAGE_
+    if not is_valid_frame_ frame: throw INVALID_UBX_CHECKSUM_
```

Comment:
If this doesn't help, we are going to need more information to figure out what bytes we see. Do we want to add this already?